mixins/providers-2.tf

```diff
@@ -0,0 +1,19 @@
+provider "aws" {
```

Comment:
It looks like this file is the same as `providers.tf` but with `iam_roles` module being two relative paths. Could we instead rename this to be something more descriptive:
 - `providers.nested.tf`
 - `providers.depth-2.tf
etc

Reply:
How about

- `providers.depth-1.tf`
- `providers.depth-2.tf`